src/test/java/org/traccar/protocol/TrvProtocolDecoderTest.java

```diff
@@ -65,6 +65,7 @@ public void testDecode() throws Exception {
         verifyPosition(decoder, text(
                 "TRVAP10080524A2232.9806N11404.9355E000.1061830323.8706000908000502,460,0,9520,3671,00,zh-cn,00"));
 
+        verifyPosition(decoder, text(
+                        "TRVYP14220217A5235.7885N00724.1840E000.0130919177.561000050660000200004,262,01,14635,52789,FritzBox7|DC-39-8F-7E-94-73|-89&FritzBox7|24-4E-5D-71-C3-9C|-90&MY_IOT|80-B4-F7-77-9C-7C|-81&MYAP|44-D4-F7-77-9C-7C|-80#"));
     }
-
```

Comment:
undo

Reply:
also re-added the other empty line according to the existing coding style

Reply:
Looks like you added some spaces here. Can you please revert so it doesn't show up as a line change in git.

Reply:
looks better now :)